jina/logging/logger.py

```diff
@@ -150,85 +129,55 @@ def close(self):
         for handler in self.logger.handlers:
             handler.close()
 
+    def add_handlers(self, config_path: str = None, **kwargs):
```

Comment:
using https://docs.python.org/3/library/logging.config.html woudln't directly work?

Reply:
not immediately as there has to be some post tweak after building the handlers. could be done after some refactoring, I will leave it as a PR for others